httpsig_test.go

```diff
@@ -607,6 +742,62 @@ func Test_Verifying_HTTP_Messages_AppendixC(t *testing.T) {
 	}
 }
 
+// Test_Verifying_HTTP_Messages_AppendixC implement tests from Appendix C
+// in the http signatures specification:
+// https://tools.ietf.org/html/draft-cavage-http-signatures-10#appendix-C
```

Comment:
See above; please remove these comments: I do not want to give the incorrect impression that this test uses keys and expected results from the draft spec.

Reply:
Yeah sorry, I should've actually read that :-)